mrtarget/common/__init__.py

```diff
@@ -214,3 +214,23 @@ def write(self,buf):
         self.buf = buf.strip('\r\n\t ')
     def flush(self):
         self.logger.log(self.level, self.buf)
+
+
+def str_to_int(s):
+    n = 0
+    try:
+        n = int(s)
+    except Exception as ex:
+        _l.exception("this should be a number but an exception was thrown " + str(ex))
+    finally:
+        return n
+
+def str_to_boolean(s):
```

Comment:
This is also dangerous. Can we make the fallback throw an exception for the caller to handle please? And document (& in a perfect world, test) the intended behaviour.